internal/clients/logger/circle_logger/circle_logger.go

```diff
@@ -0,0 +1,47 @@
+package circlelogger
```

Comment:
Can we add tests for these? Should be very similar to the `github_logger_test.go`